File: CSC242. Python II/CSC242. Python To Java/Py_Java_TempConvert.java
/*
Python CSC242
TempConvert Class()

To Execute Program run Command Prompt
> javac Py_Java_TempConvert.java
> java Py_Java_TempConvert
*/

import java.util.Scanner;

public class Py_Java_TempConvert {
	
	public static void main(String args[]){
		
		double _fahr, _cel;
		Scanner sc;
		
		sc = new Scanner(System.in);
		System.out.print("Enter Temp in F: ");
		_fahr = sc.nextDouble();
		
		_cel = (_fahr -32) * 5.0/9.0;
		
		System.out.println(String.format("Temperature in C: %.2f", _cel));
	}
}

File: CSC242. Python II/CSC242. Python To Java/Py_Java_TempConvert.py
'''
Python CSC242
TempConvert Class()
'''

class TempConvert(object):

    def main():
        _fahr = eval(input("Enter Temp in F: "))
        _cel = (_fahr -32) * 5.0/9.0

        print("Temperature in C: {:5.1f}".format(_cel))

if(__name__ == '__main__'):
    TempConvert.main()
